tests/pytype_blacklist.txt

```diff
@@ -79,3 +79,13 @@ stdlib/3/tokenize.pyi # parse only
 stdlib/3/types.pyi # parse only
 stdlib/3/urllib/error.pyi # parse only
 stdlib/3/urllib/request.pyi # parse only
+stdlib/3/collections/abc.pyi # parse only
```

Comment:
Just curious, why were these added?

Reply:
they're temporarily broken while i add some features to pytype; i just didn't want them to block this PR. i'll do a separate PR in a few days cleaning out a lot of the blacklist.